Jenkinsfile

```diff
@@ -26,7 +26,7 @@ pipeline {
           }
         }
         container('node') {
-          sh "daemon --name=sauceconnect -- /usr/local/bin/sc -u ${SAUCE_CRED_USR} -k ${SAUCE_CRED_PSW} -i ${TUNNEL_IDENTIFIER}"
+          startSauceConnect()
```

Comment:
jenkins script?

Reply:
yes, we've added the magic formula to https://github.com/molgenis/molgenis-jenkins-pipeline/ in case we need to update it again